### ISSUE TITLE
Support for placeholders & readonly tags

### ISSUE THREAD

**Issue by dertuxmalwieder** [cheunghy/jquery.colorfy#15]
Could you please consider supporting `readonly` and HTML5 placeholders for the generated `div`s? :)

**Comment by victorteokw** [comment]
placeholders should be supported.

For readonly, what's the native look of readonly textarea? Does it differ among browsers?

IMHO, regardless of style, readonly almost means toggle contenteditable.

**Comment by dertuxmalwieder** [comment]
Mostly a `readonly` input field (including textareas) looks like a normal input field (that's the difference to `disabled`). Could disabling the `contenteditable` flag be done automatically?

I tried to use `placeholder` with a `colorfy`'d `input`, but the placeholder wasn't shown. (Firefox.)

**Comment by victorteokw** [comment]
Yes it's not supported yet. I should make these features onto version 0.1.0.
Feel free to submit PR.

**Comment by dertuxmalwieder** [comment]
I made a `readonly` PR. Not sure about which way would be the best way for placeholders yet.

**Comment by victorteokw** [comment]
@dertuxmalwieder What do you think of using https://github.com/WebReflection/document-register-element to implement a custom tag? Will it cause compatibility issue for you?

**Comment by dertuxmalwieder** [comment]
The application I'm writing is basically Electron-based, so it _should_ work. **However**, I assume that this would add quite a lot of overhead and I'm not sure if the complexity wouldn't reach a level where finding implementation bugs would be too hard.
